watertap/tools/parameter_sweep/parameter_sweep_differential.py

```diff
@@ -330,6 +326,9 @@ def _run_differential_sweep(self, model, local_value):
             reinitialize_before_sweep=self.config.reinitialize_before_sweep,
             parallel_manager_class=SingleProcessParallelManager,
         )
+        # pass model_manager from refernce sweep, to diff sweep
+        # so we don't have to reijnit he model
+        diff_ps.model_manager = self.model_manager
 
         _, differential_sweep_output_dict = diff_ps.parameter_sweep(
             model,
```

Comment:
@bknueven @avdudchenko should we continue providing the model here given that we plan on deprecating passing a model in the future?